### ISSUE TITLE
Can't delete single message

### ISSUE THREAD

**Issue by thedeeno** [sj26/mailcatcher#149]
Cool project! Thanks for all the work on it.

I'm not sure if it's just me, but I'm getting the following error when trying to delete a message by ID via `delete /messages/:id`.

```
SQLite3::RangeException at /messages/2
bind or column index out of range
```

The error is occuring here:
```
def delete_message!(message_id)
  @delete_messages_query ||= db.prepare "DELETE FROM message WHERE id = ?"
  @delete_message_parts_query ||= db.prepare "DELETE FROM message_part WHERE message_id = ?"

   # the below line fails
  @delete_messages_query.execute(message_id) and
  @delete_message_parts_query.execute(message_id)
end
```

The query appears to be well formed so I'm not sure what's going on here. Anyone else getting this error?

**Comment by sj26** [comment]
Sorry not sure what's going on here. Try upgrading all your gems? SQLite had some strangeness a while back. Please re-open if you're still seeing this issue.

**Comment by dswitzer** [comment]
I was also having this issue under 0.5.2, but I can confirm the issue was resolved in 0.6.1.
